### PR TITLE
feat: Add "Be A Hacker" Section

### PR DESCRIPTION
**Pull request by sambit-sankalp**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [x] Make sure you are making a pull request against the **default branch** (left side). Also you should start *your branch* off *default branch*.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [x] I have added necessary documentation (if appropriate)

### Description
Please describe your pull request.

❤️Thank you!

### Post merge checklist
- [x] Follow steps from the [guidelines for contributing](../CONTRIBUTING.md) to this repository.
- [x] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

**Comment by riteshsp2000** [comment]
Hey @sambit-sankalp, a few changes before reviewing the PR.

1. Send all the images used in this PR to @CIPHERTron and get cloudinary links for those.
2. The register button looks cool. A few changes which I feel would be nice are, instead of making the entire button white, just make the border white color with text white. When howered on the button, you can fill it with white color and make the text black instead the current black background on shower.
3. The cards are not fully responsive and are overlapping at a few places. Also the width of the card is not adjusting as per the screen. Have a look at [project-guava-web](https://github.com/dscnitrourkela/project-guava-web), I have a similar setup which adjusts the card width.

Rest all looks good, we'll go through the code and suggest any changes if necessary

File: src/components/HackerSection/BeAHacker.jsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                    <Body>
+                        true hacker. The person who attains all these abilities will yield the power
+                    </Body>
+                    <Body>
+                        a to hack anything.
+                    </Body>
+                </Description>
+            </HackerItems>
+            <HackerItems>
+            <HackerCard />
```
Comment (riteshsp2000) [inline src/components/HackerSection/BeAHacker.jsx:43]: Indentation issue. Fix it

File: src/components/HackerSection/BeAHacker.jsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import styled from 'styled-components';
+import tw from 'twin.macro';
+import { Container, SectionContainer } from '../shared';
+import Heading from '../shared/Typography/Heading2';
+import Body from '../shared/Typography/Body';
+import HackerCard from "./HackerCard";
+
```
Comment (riteshsp2000) [inline src/components/HackerSection/BeAHacker.jsx:8]: We use 2 space indentation. All of your code is with 4 space indentation.

**Review by riteshsp2000 — CHANGES_REQUESTED** [review]

**Review by CIPHERTron — APPROVED** [review]

**Review by riteshsp2000 — APPROVED** [review]

**Review by abhibhaw — APPROVED** [review]

**Comment by riteshsp2000** [comment]
@all-contributors add @sambit-sankalp for Code

**Comment by CIPHERTron** [comment]
@all-contributors add @sambit-sankalp for Code

**Pull request merged**